CRM/Civicase/Hook/BuildForm/TokenTree.php

```diff
@@ -479,7 +479,7 @@ private function shouldRun($formName) {
     return CRM_Utils_Request::retrieve('caseid', 'Integer') &&
       in_array(
         $formName,
-        [CRM_Contact_Form_Task_Email::class, CRM_Contact_Form_Task_PDF::class]
+        [CRM_Case_Form_Task_Email::class, CRM_Contact_Form_Task_PDF::class]
```

Comment:
same here, `CRM_Contact_Form_Task_PDF` probably need to be changed to `CRM_Case_Form_Task_PDF`, in general you will need to search the whole extension for `CRM_Contact_Form_Task_PDF`